src/platform/user/profile/utilities/index.js

```diff
@@ -126,11 +128,14 @@ export function mapRawUserDataToState(json) {
 // as a trigger to properly update any components that subscribe to it.
 export const hasSession = () => localStorage.getItem('hasSession');
 
+export const hasSessionSSO = () => localStorage.getItem('hasSessionSSO');
+
 export function setupProfileSession(userProfile) {
   const { firstName, signIn } = userProfile;
   const loginType = (signIn && signIn.serviceName) || null;
 
   localStorage.setItem('hasSession', true);
+  ssoKeepAliveSession();
```

Comment:
@U-DON is this the right place to initialize the localstorage flag for SSO? I flip-flopped on this basically because there are 2 cases:

1. A user who logs in via VA.gov and subsequently has an active SSO session
2. A user who has an active SSO session from another VA website but has not yet authenticated via VA.gov should have their session continuity/extension still maintained. I believe the UX here would still require them to click "sign-in" (and just not have to re-enter their credentials) before their VA.gov session is officially initiated

Reply:
This is definitely the place to cover case 1.

If a user is already signed in, there is a request to `/user` on every page load from the user header component, so that should set the flag, and you might not need to add anything else.

Reply:
Perfect. I think case 2 might be covered by default server-side. If there's an active SSOe session and we hit the sessions v1 endpoint, it should be cached on the SSOe side and propagate through to us, just like clicking "sign-in" on other SSO-style sites. 

It's possible we'll need to make tiny changes to facilitate that model, but it shouldn't block these changes from merging